DNN Platform/Modules/DigitalAssets/ClientScripts/dnn.DigitalAssets.js

```diff
@@ -2573,21 +2573,51 @@ dnnModule.digitalAssets = function ($, $find, $telerik, dnnModal) {
         }
     }
 
+    function listItemsToDelete(itemsToDelete, outputStr) {
+        for (var y = 0; y < itemsToDelete.length; y++) {
+            var delimiter = ", ";
+            if (y == (itemsToDelete.length - 1)) {
+                delimiter = "";
+            }
+
+            outputStr += itemsToDelete[y].ItemName + delimiter;
+        }
+        return outputStr;
+    }
+
     function confirmDeleteItems(items, parentFolderId, mappedSubfoldersCount) {
         var folderAndFileText = selectionText(items);
         var dialogTitle = resources.deleteTitle.replace('[ITEMS]', folderAndFileText);
         var dialogText = resources.deleteConfirmText.replace('[ITEMS]', folderAndFileText);
 
         var dialogNote = "";
-        var dialogHeight = 190;
+        var dialogHeight = 260;
```

Comment:
we likely need to allow the dialog to set the height based on the content. Now the list of files and folders to be deleted could be potencially be very long.